### DISCUSSION TITLE
Automatic capitalisation

### DISCUSSION THREAD

**Discussion by rhhsm** [streetcomplete/StreetComplete#4958] [Q&A] [answered]
I noticed that when entering names (street names, for instance) in SC, each word is automatically capitalised on my phone. Is this something that SC does, or is it done by my keyboard app (Microsoft Swiftkey), by my phone (LG Q60) or maybe it's an Android 10 thing? This capitalisation is OK for English, but wrong for Bulgarian (a common street name is named after Stara Planina/Стара планина, but Стара планина becomes Стара Планина as soon as it's followed by a space, which is wrong). It's a bit irritating to have to go back often and correct the capitalisation of the Bulgarian entry. Where should I complain about this? :)

**Comment by riQQ** [comment, marked as answer]
Related to this recent issue: #4784

**Reply by rhhsm** [reply]
Thanks, I missed that issue. Will mention it there

**Reply by matkoniecz** [reply]
better open new one

**Reply by westnordost** [reply]
I reopened that issue because apparently using the titlecase function didn't actually fix it.
